src/containerapp/azext_containerapp/tests/latest/test_containerappjob_preview_scenario.py

```diff
@@ -0,0 +1,147 @@
+            self.cmd(f'aks get-credentials --resource-group {resource_group} --name {aks_name} --overwrite-existing --admin')
+
+            self.cmd(f'connectedk8s connect --resource-group {resource_group} --name {connected_cluster_name}')
+            connected_cluster = self.cmd(f'az connectedk8s show --resource-group {resource_group} --name {connected_cluster_name}').get_output_in_json()
+
+            connected_cluster_id = connected_cluster.get('id')
+            extension = self.cmd(f'az k8s-extension create'
+                                 f' --resource-group {resource_group}'
+                                 f' --name containerapp-ext'
+                                 f' --cluster-type connectedClusters'
+                                 f' --cluster-name {connected_cluster_name}'
+                                 f' --extension-type "Microsoft.App.Environment" '
+                                 f' --release-train stable'
+                                 f' --auto-upgrade-minor-version true'
+                                 f' --scope cluster'
+                                 f' --release-namespace appplat-ns'
+                                 f' --configuration-settings "Microsoft.CustomLocation.ServiceAccount=default"'
+                                 f' --configuration-settings "appsNamespace=appplat-ns"'
+                                 f' --configuration-settings "clusterName={connected_cluster_name}"'
+                                 f' --configuration-settings "envoy.annotations.service.beta.kubernetes.io/azure-load-balancer-resource-group={resource_group}"').get_output_in_json()
+            custom_location_name = "my-custom-location"
+            custom_location_id = self.cmd(f'az customlocation create -g {resource_group} -n {custom_location_name} -l {TEST_LOCATION} --host-resource-id {connected_cluster_id} --namespace appplat-ns -c {extension["id"]}').get_output_in_json()['id']
+        except Exception as e:
+            pass
+
+        # create connected environment with client or create a command for connected?
+        sub_id = self.cmd('az account show').get_output_in_json()['id']
+
+        connected_env_name = 'my-connected-env'
+        connected_env_resource_id = f"/subscriptions/{sub_id}/resourceGroups/{resource_group}/providers/Microsoft.App/connectedEnvironments/{connected_env_name}"
+        file = f"{resource_group}.json"
+        env_payload = '{{ "location": "{location}", "extendedLocation": {{ "name": "{custom_location_id}", "type": "CustomLocation" }}, "Properties": {{}}}}' \
+            .format(location=TEST_LOCATION, custom_location_id=custom_location_id)
+        write_test_file(file, env_payload)
+        self.cmd(f'az rest --method put --uri "{connected_env_resource_id}?api-version=2022-06-01-preview" --body "@{file}"')
```

Comment:
we do not have connected env commands in extension?

Reply:
Not yet. I will move connected env commands in next PR.